app/routes/index/mygoals.js

```diff
@@ -1,8 +1,20 @@
 import Ember from 'ember';
 import AuthenticatedRouteMixin from 'feedbackfruits-accounts/mixins/authenticated-route-mixin';
 
+//const { inject: { service } } = Ember;
+
 export default Ember.Route.extend(AuthenticatedRouteMixin, {
+//  account: service(),
   model(){
+  /*
```

Comment:
This code looks cool, why is it commented out? Anyway, if its not useful, I think it should be removed.